### PR TITLE
docs(QTM-698): Inclusion in the docs of the suggestion not to use MUI in implem

### PR DESCRIPTION
**Pull request by MarcosViniciusPC**
## Description
https://jirasoftware.catho.com.br/browse/QTM-698

## Review guide
- [ ] Doc

### CODE REVIEW

**Review by rapahaeru — DISMISSED** [review]

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -26,7 +27,15 @@ Obs.: We have .nvmrc. When you run nvm use, it will set the node to the same ver
 Why _commitizen_? To enforce a padronization to the commits and make sure every commit follows the same profile.
 
 ---
+## Important Notices ⚠️ 
 
+We do not recommend using ready-made components from [MUI](https://mui.com/material-ui/getting-started/) when implementing Quantum components for performance and long-term maintenance reasons. MUI is included in our library only to enable the use of [Material Icons](https://mui.com/material-ui/material-icons/).
+
+The MUI library, as well as the Styled Components, internally use resources and libraries that are an impediment to the use of [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource included in Next JS 13 that we intend to enable its use and consequently improve the performance of our components. In the future, we plan to remove the cited libraries completely.
```
Comment (alizeleal) [inline CONTRIBUTING.MD:34]: The MUI library, as well as the Styled Components, uses internally resources and libraries that aren't compatible with [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource added to Next JS 13. We intend to enable its use in the future to improve the performance of our components. Therefore, we plan to remove the mentioned libraries completely from our components.

Comment (alizeleal) [inline CONTRIBUTING.MD:34]: The MUI library, as well as the Styled Components, uses internally resources and libraries that aren't compatible with [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource added to Next JS 13. We intend to enable its use in the future to improve the performance of our components. Therefore, we plan to remove the mentioned libraries completely from our components.

File: CONTRIBUTING.MD [hunk 2]
```diff
@@ -26,7 +27,15 @@ Obs.: We have .nvmrc. When you run nvm use, it will set the node to the same ver
...
+
+We also do not recommend using any other ready-made component libraries. We suggest that new components have their DOM structure, styling and functionalities completely implemented internally, that is, without dependence on third-party libraries.
```
Comment (alizeleal) [inline CONTRIBUTING.MD:36]: Also, we do not recommend using any other ready-made component library. We suggest for new components to have DOM structure, styling and functionalities completely implemented internally, without any dependence on third-party libraries.

File: README.md [hunk 3]
```diff
@@ -12,6 +12,14 @@
 > **Warning**
 > Quantum requires react@16.14.0 or higher.
 
+## Notices ⚠️
+
+For performance and long-term maintenance reasons, we do not recommend using [MUI](https://mui.com/material-ui/getting-started/) in Quantum component implementations. MUI is included in our library only to enable the use of [Material Icons](https://mui.com/material-ui/material-icons/).
```
Comment (alizeleal) [inline README.md:17]: For performance and long-term maintenance reasons, we do not recommend using [MUI](https://mui.com/material-ui/getting-started/) in Quantum components implementation. MUI is present in our library only to allow the use of [Material Icons](https://mui.com/material-ui/material-icons/).

File: README.md [hunk 4]
```diff
@@ -12,6 +12,14 @@
...
+
+We also do not recommend using any other ready-made component libraries. We suggest that new components have their DOM structure, styling and functionalities completely implemented internally, that is, without dependence on third-party libraries.
```
Comment (alizeleal) [inline README.md:19]: Also, we do not recommend using any other ready-made component library. We suggest for new components to have DOM structure, styling and functionalities completely implemented internally, without any dependence on third-party libraries.

**Review by alizeleal — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.MD (hunk 1 above)
Comment (alizeleal) [inline CONTRIBUTING.MD:34]:
```suggestion
The MUI library, as well as the Styled Components, uses internally resources and libraries that aren't compatible with [Server Components](https://nextjs.org/docs/app/building-your-application/rendering/server-components), a resource added to Next JS 13. We intend to enable its use in the future to improve the performance of our components. Therefore, we plan to remove the mentioned libraries completely from our components.
```

File: CONTRIBUTING.MD (hunk 2 above)
Comment (alizeleal) [inline CONTRIBUTING.MD:36]:
```suggestion
Also, we do not recommend using any other ready-made component library. We suggest for new components to have DOM structure, styling and functionalities completely implemented internally, without any dependence on third-party libraries.
```

File: README.md (hunk 3 above)
Comment (alizeleal) [inline README.md:17]:
```suggestion
For performance and long-term maintenance reasons, we do not recommend using [MUI](https://mui.com/material-ui/getting-started/) in Quantum components implementation. MUI is present in our library only to allow the use of [Material Icons](https://mui.com/material-ui/material-icons/).
```

File: README.md (hunk 4 above)
Comment (alizeleal) [inline README.md:19]:
```suggestion
Also, we do not recommend using any other ready-made component library. We suggest for new components to have DOM structure, styling and functionalities completely implemented internally, without any dependence on third-party libraries.
```

**Review by alizeleal — CHANGES_REQUESTED** [review]

**Review by alizeleal — APPROVED** [review]

**Pull request merged**